### PR TITLE
Check Arc perf test total results

### PR DESCRIPTION
**Pull request by WeiguangHan**
@hkvision Fix the visualization of highlighted values in html of arc perf test.

### CODE REVIEW

File: python/llm/test/benchmark/check_html_lines.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#
```
Comment (hkvision) [inline python/llm/test/benchmark/check_html_lines.py:1]: rename to `check_results.py`

File: python/llm/test/benchmark/check_html_lines.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# Python program to check if the number of lines in html meets expectation
+
+import os
+import sys
+import argparse
+import pandas as pd
+
+def main():
+    parser = argparse.ArgumentParser(description="check if the number of lines in html meets expectation")
+    parser.add_argument("-f", "--folder_path", type=str, dest="folder_path",
```
Comment (hkvision) [inline python/llm/test/benchmark/check_html_lines.py:26]: I think you only need to input one file?
And you can directly use the raw csv file to do so in the original folder?

File: python/llm/test/benchmark/check_html_lines.py (hunk 1 above)
Comment (WeiguangHan) [reply python/llm/test/benchmark/check_html_lines.py:1]: sure

File: python/llm/test/benchmark/check_html_lines.py (hunk 2 above)
Comment (WeiguangHan) [reply python/llm/test/benchmark/check_html_lines.py:26]: ok. Will change this according to your suggestions.

**Pull request closed without merging**